evap/results/views.py

```diff
@@ -135,5 +135,12 @@ def user_can_see_text_answer(user, text_answer, public_view=False):
             return True
         if text_answer.contribution.course.is_user_responsible_or_delegate(user):
             return True
```

Comment:
these two lines should be superfluous now if i got that correctly. also, please put `list(user.represented_users.all()) into a variable to avoid hitting the database twice in the common case (which is (i think) that a comment is not visible)